### PR TITLE
10174 add auth_ldap_self_signed module

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: auth_ldap_self_signed/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Authentication via LDAP with self signed certificate',
+    'version': '0.1',
```
Comment (siyuancn) [inline auth_ldap_self_signed/__openerp__.py]: wrong version

File: auth_ldap_self_signed/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    'category': '',
```
Comment (siyuancn) [inline auth_ldap_self_signed/__openerp__.py]: please put the category, otherwise remove the empty line

**Comment by siyuancn** [comment]
Please user the structure:
A module is organised in a few directory:

```
controllers/: contains controllers (http routes)
data/: data xml
demo/: demo xml
models/: models definition
report/: reporting models (BI/analysis), Webkit/RML print report templates
static/: contains the web assets, separated into css/, js/, img/, lib/, ...
views/: contains the views and templates, and QWeb report print templates
wizards/: wizard model and views
```

File: auth_ldap_self_signed/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Credits
+=======
+
+
+Contributors
+------------
+
+* Augustin Cisterne-Kaas
+
+Maintainer
```
Comment (elicoidal) [inline auth_ldap_self_signed/README.rst:50]: Elico

Comment (Rona111) [inline auth_ldap_self_signed/README.rst:50]: fixed.@elicoidal @seb-elico @duanyp1991 @siyuancn

**Pull request merged**